website/content/docs/auth/azure.mdx

```diff
@@ -56,6 +56,15 @@ or [`resource_id`](/vault/api-docs/auth/azure#resource_id) parameters are used o
 
 ## Authentication
 
+### Root Credentials API Permissions
```

Comment:
I'm realizing now that the "Authentication" section here has to do with actually authenticating (i.e., logging in) to via the plugin. I think this recommendation around API permissions is out of place here.

I suggest to place this in the above section with "Prerequisites". Perhaps we can add a "###" level section called "Role Assignments" on line 41 and a "###" level section called "API Permissions" on line 56 that has the content you've added below here. Let me know if that doesn't make sense 🙂

Reply:
That makes sense. I just moved it under the Prerequisites and added new sections. Let me know if this is not what you had in mind 😃 
Thank you!